tests/unit/test_charm.py

```diff
@@ -507,11 +507,11 @@ def test_reconfigure_remove_member_failure(self, connection, defer):
             )
 
             # simulate 2nd MongoDB unit joining( need a unit to join before removing a unit)
-            self.harness.add_relation_unit(rel.id, "mongodb/1")
-            self.harness.update_relation_data(rel.id, "mongodb/1", PEER_ADDR)
+            self.harness.add_relation_unit(rel.id, "mongodb-k8s/1")
+            self.harness.update_relation_data(rel.id, "mongodb-k8s/1", PEER_ADDR)
 
             # simulate removing 2nd MongoDB unit
-            self.harness.remove_relation_unit(rel.id, "mongodb/1")
+            self.harness.remove_relation_unit(rel.id, "mongodb-k8s/1")
```

Comment:
Not worth adding in this PR, but worth adding a global `CHARM_KEY` or similar to avoid this.

Reply:
separate PR should be good, thanks